go/vt/vtctl/vtctl.go

```diff
@@ -323,10 +323,10 @@ var commands = []commandGroup{
 				"[-cells=c1,c2,...] [-reverse] <destination keyspace/shard> <served tablet type>",
 				"Makes the <destination keyspace/shard> serve the given type. This command also rebuilds the serving graph."},
 			{"MigrateReads", commandMigrateReads,
-				"[-cells=c1,c2,...] [-reverse] -workflow=workflow <target keyspace> <tablet type>",
+				"[-cells=c1,c2,...] [-reverse] -tablet_type={replica|rdonly} <keyspace.workflow>",
```

Comment:
Does this have to be done one tablet_type at a time?
Also, consistency? VDiff has `-tablet_types` and this command has `-tablet_type`.

Reply:
This is inherited from the older `Migrate` behavior where we allow only one tablet type per command. I've been thinking of making this more flexible, but we need to brainstorm that part before committing. For example, how about allowing `MigrateWrites` before `MigrateReads`?

As for vdiff, `tablet_types` means something else. It means choose any of those tablet types to compare. So, you could also specify `master` in that case.